docs/machine-learning/index.md

```diff
@@ -0,0 +1,33 @@
+---
+title: ML.NET Guide
+description: Learn how to build custom AI solutions and integrate them into your .NET applications using ML.NET.
+author: aditidugar
+ms.date: 05/07/2018
+---
+# ML.NET Guide
+
+ML.NET is a free, open-source, and cross-platform machine learning framework that enables you to build custom machine learning solutions and integrate them into your .NET applications. This guide provides many resources about working with ML.NET.
+
+For more information about ML.NET, see [ntroducing ML.NET: Cross-platform, Proven and Open Source Machine Learning Framework ](https://blogs.msdn.microsoft.com/dotnet/) post on the .NET blog.
```

Comment:
Typo in the link text: *ntroducing ML.NET...*
In general, that is great, great addition!!!